src/main/java/seedu/planner/model/course/FocusArea.java

```diff
@@ -25,6 +27,35 @@
         this.name = name;
     }
 
+    /**
+     * Check if a string matches a {@code FocusArea} enum value.
+     *
+     * Note: Case-insensitive
+     */
+    private static boolean hasFocusArea(String focusArea) {
+        focusArea = focusArea.toLowerCase();
+        for (FocusArea fa : FocusArea.values()) {
+            if (fa.toString().toLowerCase().equals(focusArea)) {
```

Comment:
Can use the `StringUtil#areEqualIgnoreCase` method if you want

Reply:
I use `.toLowerCase()` directly so that `focusArea` will only be lower-cased once (avoid unneeded memory allocation).